.vscode/tasks.json

```diff
@@ -4,31 +4,19 @@
     {
       "label": "Install requirements",
       "type": "shell",
-      "command": "pip3 install -r requirements-first.txt;pip3 install -r requirements-dev.txt",
+      "command": "pip3 install -r requirements.txt;pip3 install -r requirements-dev.txt",
```

Comment:
I think just requirements-dev.txt is enough, as the requirements in requirements.txt are already included in there.  Years ago there used to be a version mismatch between the pycryptodome library required for tinytuya and one of the other dependencies, so we used to install that first to ensure the correct version was installed, but the versions are in sync now which is why requirements-first.txt was removed to simplify the process.

Reply:
Thanks, I'll update this when I back from a work trip.